core/bindings.rs

```diff
@@ -222,7 +225,10 @@ fn op_ctx_function<'s>(
   } else {
     builder.build(scope)
   };
-  templ.get_function(scope).unwrap()
+
+  let v8fn = templ.get_function(scope).unwrap();
```

Comment:
**question:** `FunctionTemplate` also has a `SetClassName` method. Would that do the same thing?

Reply:
I'm not sure, but it wouldn't work in this case, because we use `FunctionTemplate::builder_raw()` which returns an instance of `FunctionBuilder` which doesn't have this API.